### ISSUE TITLE
Code Cell in Jupyter

### ISSUE THREAD

**Issue by bsstertius** [mwermelinger/Learn-to-code-for-data-analysis#12]
I cannot write to the code cell whilst working through the Jupyter Exercise book. What am I doing wrong ?  Thanks in anticipation

**Comment by mwermelinger** [comment]
I never heard that error reported for these notebooks. Do you mean you can’t edit any code cell, it doesn’t even allow you to type in the cell?

**Comment by bsstertius** [comment]
Thats right. The tutorial says that clicking to the left of the cell allows
editing within the cell - and the border changes colour (?). None of this
happens in the note book I have on screen.

I wonder if I somehow have a copy of the notebook rather than one intended
for student use.

Bernard

**Comment by mwermelinger** [comment]
Sorry for the delay replying. Jupyter has two modes: edit (green cell border) and command (blue border). Clicking on the left selects the cell and switches to command mode. Clicking in the cell or pressing ENTER switches to edit mode. Pressing on ESC also switches to command mode.

All notebooks are for student use, there’s no separate read-only copy. HTH.

**Comment by bsstertius** [comment]
Thanks for getting back to me. I can report that today the cells in Jupiter
notebooks are behaving as expected. Frustratingly, I don't know that I did
anything different - but at least I can now be confident that the
installation is OK.

Best wishes

Bernard
